.github/actions/curl-meta-yaml/action.yaml

```diff
@@ -10,5 +10,6 @@ runs:
         curl -s -L -o conda-recipe/meta.yaml https://raw.githubusercontent.com/conda-forge/genomekit-feedstock/main/recipe/meta.yaml
         # build from local source, not from the latest release tarball
         sed -i -e 's|url: https://github.com/deepgenomics/GenomeKit/archive/refs/tags/v.*$|path: ../|1; /sha256: /d' conda-recipe/meta.yaml
-        head -10 conda-recipe/meta.yaml
+        sed -i -e 's|python -m unittest discover|python -m unittest discover -v -b|1' conda-recipe/meta.yaml
```

Comment:
so we still want to run the import tests but not the unit tests?  maybe a comment here would be helpful